### PR TITLE
Why

### PR DESCRIPTION
**Pull request by Scrxtchy**
image

### CODE REVIEW

**Comment by nokonoko** [comment]
Will look thru it tomorrow, thanks!

**Comment by Scrxtchy** [comment]
Still working on it lmao

**Comment by nokonoko** [comment]
Dope

**Comment by Scrxtchy** [comment]
I thought I actually hit the draft button, looks like I missed it. Oh Well, shit's good now. Can get some more visual work flare but honestly this is good progress

**Comment by nokonoko** [comment]
U did hit draft lmao, I accidentally hit ready to review. Is it ready for being merged and take for a spin? Also visually there is a lot that has to be done like not use entire bootstrap, not having the text in code blocks etc. But for 2 days work this is great so far.

**Comment by Scrxtchy** [comment]
Yeah shit seems to be working fine here, anything new would be on a new PR :shipit:

File: includes/js/board.js [hunk 1]
```diff
@@ -10,4 +10,49 @@ window.onload = () => {
...
   })
 })()
+  this.setInterval(fetchNewPosts, 30*1000)
 }
+
+
+async function fetchNewPosts(){
+  //this will need some look into later
+  //as your DOM looks like needs work, this can be adjusted later
+  console.log("Fetching")
```
Comment (Scrxtchy) [inline includes/js/board.js:20]: I should have removed this line

**Pull request merged**